app/controllers/citizens/other_assets_controller.rb

```diff
@@ -18,7 +18,7 @@ def update
     private
 
     def declaration
-      @declaration ||= legal_aid_application.other_assets_declaration
+      @declaration ||= legal_aid_application.other_assets_declaration || legal_aid_application.build_other_assets_declaration
```

Comment:
Another bug fix - second time I've fixed this, so probably a merge problem.

Reply:
I just fixed it and merged it with this PR: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/351

The other day you did it on the provider's journey of this page: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/340/files#diff-264e6f23d8630656383e22b5287e45ff